relayer/message_relayer.go

```diff
@@ -64,22 +64,19 @@ type messageRelayer struct {
 }
 
 func newMessageRelayer(
-	logger logging.Logger,
-	metrics *MessageRelayerMetrics,
 	relayer *Relayer,
 	warpMessage *warp.UnsignedMessage,
 	destinationBlockchainID ids.ID,
 	messageResponseChan chan message.InboundMessage,
-	messageCreator message.Creator,
 ) *messageRelayer {
 	return &messageRelayer{
 		relayer:                 relayer,
 		warpMessage:             warpMessage,
 		destinationBlockchainID: destinationBlockchainID,
 		messageResponseChan:     messageResponseChan,
```

Comment:
We can also reference `relayer.responseChan` directly instead of passing it as a separate argument.